### PR TITLE
Fix signup page crash

### PR DESCRIPTION
**Pull request by DukeManh**
This fixes the signup page crash due to `useEffect` infinite loop in 'SignUp.tsx'
## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -9,11 +9,13 @@ tasks:
       eval $(gp env -e API_URL=$(gp url 3000))
       eval $(gp env -e WEB_URL=$(gp url 8000))
       eval $(gp env -e API_HOST=$(gp url 8443))
+      eval $(gp env -e SSO_URL=$(gp url 8081))
```
Comment (humphd) [inline .gitpod.yml:12]: I don't think this file should be changing as part of this PR, should it?

**Review by humphd — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (DukeManh) [reply .gitpod.yml:12]: I wanted to make the singup flow works for people who might review this PR using Gitpod.

File: .gitpod.yml [hunk 2]
```diff
@@ -9,11 +9,13 @@ tasks:
...
       eval $(gp env -e SERVICES="status image auth search posts feed-discovery users redis elasticsearch login firebase")
 
       sed -r \
           -e  "s@(.+=)http://localhost:8000(/[^ ]*)*@\1$WEB_URL\2@g" \
           -e  "s@(.+=)http://localhost:3000(/[^ ]*)*@\1$API_URL\2@g" \
+          -e  "s@(.+=)http://localhost:8081(/[^ ]*)*@\1$SSO_URL\2@g" \
```
Comment (humphd) [inline .gitpod.yml:18]: I was just surprised to see this change, since it is based on my other PR, right?

**Review by humphd — DISMISSED** [review]

**Review by HyperTHD — DISMISSED** [review]

**Comment by humphd** [comment]
This needs a rebase and can go in after that merge conflict is fixed.

File: .gitpod.yml (hunk 2 above)
Comment (DukeManh) [reply .gitpod.yml:18]: No, I just happened to have the same idea.

**Review by humphd — APPROVED** [review]

**Review by humphd — APPROVED** [review]

**Review by aserputov — APPROVED** [review]

**Comment by aserputov** [comment]
@humphd can we merge this one ?

**Comment by humphd** [comment]
@aserputov go for it, click `Rebase and merge`.  As long as it doesn't have merge conflicts, and isn't more than 1 commit, it's fine with 2 approvals.

**Pull request merged**